### ISSUE TITLE
Error: The events file C:\...\events.out.tfevents.1512313073.DESKTOP-5TN0RO2 has disappeared.

### ISSUE THREAD

**Issue by MartinGeierino** [rstudio/tensorflow#208]
Hi.

I'm having this weird "error". (Doesn't really throw an error, but writes in red text to console and bugs the rest of my output.)

I'm running DNN regressor like this:

```
userID <- column_categorical_with_vocabulary_list("userID", vocabulary_list = unique(mTrain$userID))
itemID <- column_categorical_with_vocabulary_list("itemID", vocabulary_list = unique(mTrain$itemID))

deep_columns <- feature_columns(column_embedding(categorical_column = userID, dimension = 128),
                                column_embedding(categorical_column = itemID, dimension = 128))

model <- dnn_regressor(feature_columns = deep_columns, 
                       hidden_units = c(256,128,64,32))

#### Training
# Input
train_input_fn <- input_fn(object = mTrain, 
                           features = c("userID", "itemID"), 
                           response = "score", 
                           batch_size = 512, 
                           shuffle = "auto", 
                           num_epochs = 1)

eval_input_fn <- input_fn(object = mVal, 
                          features = c("userID", "itemID"), 
                          response = "score", 
                          batch_size = 512, 
                          shuffle = "auto", 
                          num_epochs = 1)

# TrainLoop über Epochs
numEpochs <- 10

for(i in 1:numEpochs){
  train(model, train_input_fn)
  
  simplifyTEST <- function(x){print(paste("EPOCH ",i," TEST Loss: ",x$loss," Avg_Loss: ",x$average_loss))}
  simplifyTRAIN <- function(x){print(paste("EPOCH ",i,"TRAIN Loss: ",x$loss," Avg_Loss: ",x$average_loss))}
  
  trainEval <- evaluate(model, train_input_fn, simplify = simplifyTRAIN)
  testEval <- evaluate(model, eval_input_fn, simplify = simplifyTEST)
  
}
```

This runs with  expected output until Epoch 5, then prints some error messages in red, and then some bugged evaluation output(The bold text is printed in red. Everythin below it gets printed without newlines):

> [/] Evaluating -- loss: 10.39, step: 744[1] "EPOCH  1 TRAIN Loss:  92.0055313110352  Avg_Loss:  0.179916515946388"
> [/] Evaluating -- loss: 82.66, step: 16[1] "EPOCH  1  TEST Loss:  99.3653564453125  Avg_Loss:  0.196131959557533"
> [/] Evaluating -- loss: 4.66, step: 744[1] "EPOCH  2 TRAIN Loss:  71.2760696411133  Avg_Loss:  0.139380127191544"
> [/] Evaluating -- loss: 70.10, step: 16[1] "EPOCH  2  TEST Loss:  93.2095489501953  Avg_Loss:  0.183981344103813"
> [/] Evaluating -- loss: 5.08, step: 744[1] "EPOCH  3 TRAIN Loss:  51.7415046691895  Avg_Loss:  0.101180344820023"
> [/] Evaluating -- loss: 84.25, step: 16[1] "EPOCH  3  TEST Loss:  93.7468566894531  Avg_Loss:  0.185041904449463"
> [/] Evaluating -- loss: 4.16, step: 744[1] "EPOCH  4 TRAIN Loss:  44.6376686096191  Avg_Loss:  0.0872888118028641"
> [/] Evaluating -- loss: 78.69, step: 16[1] "EPOCH  4  TEST Loss:  99.4734268188477  Avg_Loss:  0.196345284581184"
> [/] Evaluating -- loss: 1.64, step: 744[1] "EPOCH  5 TRAIN Loss:  37.0394439697266  Avg_Loss:  0.0724305137991905"
> [/] Evaluating -- loss: 75.64, step: 16[1] "EPOCH  5  TEST Loss:  107.577430725098  Avg_Loss:  0.212341338396072"
> [/] Training -- loss: 0.79, step: 744 **The events file C:\Users\Martin\AppData\Local\Temp\tmpxdplggex\events.out.tfevents.1512313073.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:00:23.378395: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 64 events to C:\Users\Martin\AppData\Local\Temp\tmpxdplggex\events.out.tfevents.1512313073.DESKTOP-5TN0RO2
> 2017-12-03 16:01:55.354496: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpxdplggex\events.out.tfevents.1512313073.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:01:55.354513: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 116 events to C:\Users\Martin\AppData\Local\Temp\tmpxdplggex\events.out.tfevents.1512313073.DESKTOP-5TN0RO2
> 2017-12-03 16:02:06.661773: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpxdplggex\events.out.tfevents.1512313073.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:02:06.661790: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 128 events to C:\Users\Martin\AppData\Local\Temp\tmpxdplggex\events.out.tfevents.1512313073.DESKTOP-5TN0RO2
> 2017-12-03 16:03:26.762138: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:03:26.762162: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 20 events to C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2
> 2017-12-03 16:04:07.544331: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:04:07.544346: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 40 events to C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2
> 2017-12-03 16:04:25.908420: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:04:25.908450: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 41 events to C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2
> 2017-12-03 16:04:48.747638: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:04:48.747654: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 60 events to C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2
> 2017-12-03 16:05:30.457953: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 16:05:30.457969: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 80 events to C:\Users\Martin\AppData\Local\Temp\tmpgdndjlom\events.out.tfevents.1512313338.DESKTOP-5TN0RO2**
> [\] Evaluating -- loss: 35.30, step: 2[|] Evaluating -- loss: 28.44, step: 3[/] Evaluating -- loss: 38.34, step: 4[-] Evaluating -- loss: 34.78, step: 5[\] Evaluating -- loss: 34.01, step: 6[|] Evaluating -- loss: 35.55, step: 7[/] Evaluating -- loss: 35.56, step: 8[-] Evaluating -- loss: 42.38, step: 9[\] Evaluating -- loss: 39.13, step: 10[|] Evaluating -- loss: 31.82, step: 11[/] Evaluating -- loss: 44.38, step: 12[-] Evaluating -- loss: 39.94, step: 13[\] Evaluating -- loss: 32.82, step: 14[|] Evaluating -- loss: 40.02, step: 15[/] Evaluating -- loss: 37.52, step: 16[-] Evaluating -- loss: 31.97, step: 17[\] Evaluating -- loss: 36.87, step: 18[|] Evaluating -- loss: 37.11, step: 19[/] Evaluating -- loss: 37.75, step: 20[-] Evaluating -- loss: 39.54, step: 21[\] Evaluating -- loss: 29.55, step: 22[|] Evaluating -- loss: 41.98, step: 23[/] Evaluating -- loss: 29.87, step: 24[-] Evaluating -- loss: 27.99, step: 25[\] Evaluating -- loss: 34.14, step: 26[|] Evaluating -- loss: 30.64, step: 27[/] Evaluating -- loss: 41.49, step: 28[-] Evaluating -- loss: 33.49, step: 29[\] Evaluating -- loss: 35.40, step: 30[|] Evaluating -- loss: 39.10, step: 31[/] Evaluating -- loss: 34.37, step: 32[-] Evaluating -- loss: 34.52, step: 33[\] Evaluating -- loss: 36.55, step: 34[|] Evaluating -- loss: 33.77, step: 35[/] Evaluating -- loss: 37.94, step: 36[-] Evaluating -- loss: 36.70, step: 37[\] Evaluating -- loss: 40.08, step: 38[|] Evaluating -- loss: 30.49, step: 39[/] Evaluating -- loss: 32.37, step: 40[-] Evaluating -- loss: 42.86, step: 41[\] Evaluating -- loss: 33.10, step: 42[|] Evaluating -- loss: 35.11, step: 43[/] Evaluating -- loss: 36.52, step: 44[-] Evaluating -- loss: 31.81, step: 45[\] Evaluating -- loss: 26.81, step: 46[|] Evaluating -- loss: 36.63, step: 47[/] Evaluating -- loss: 36.10, step: 48[-] Evaluating -- loss: 39.69, step: 49[\] Evaluating -- loss: 40.49, step: 50[|] Evaluating -- loss: 37.77, step: 51[/] Evaluating -- loss: 30.01, step: 52[-] Evaluating -- loss: 32.79, step: 53[\] Evaluating -- loss: 32.33, step: 54[|] Evaluating -- loss: 37.64, step: 55[/] Evaluating -- loss: 39.76, step: 56[-] Evaluating -- loss: 38.06, step: 57[\] Evaluating -- loss: 37.22, step: 58[|] Evaluating -- loss: 25.31, step: 59[/] Evaluating -- loss: 41.79, step: 60[-] Evaluating -- loss: 41.19, step: 61[\] Evaluating -- loss: 40.35, step: 62[|] Evaluating -- loss: 27.30, step: 63[/] Evaluating -- loss: 30.01, step: 64[-] Evaluating -- loss: 34.41, step: 65[\] Evaluating -- loss: 27.90, step: 66[|] Evaluating -- loss: 36.74, step: 67[/] Evaluating -- loss: 35.30, step: 68[-] Evaluating -- loss: 38.21, step: 69[\] Evaluating -- loss: 34.40, step: 70[|] Evaluating -- loss: 33.18, step: 71[/] Evaluating -- loss: 39.54, step: 72[-] Evaluating -- loss: 35.63, step: 73[\] Evaluating -- loss: 33.07, step: 74[|] Evaluating -- loss: 32.47, step: 75[/] Evaluating -- loss: 29.82, step: 76[-] Evaluating -- loss: 34.47, step: 77[\] Evaluating -- loss: 37.35, step: 78[|] Evaluating -- loss: 36.10, step: 79[/] Evaluating -- loss: 28.19, step: 80[-] Evaluating -- loss: 35.15, step: 81[\] Evaluating -- loss: 35.05, step: 82[|] Evaluating -- loss: 32.54, step: 83[/] Evaluating -- loss: 36.78, step: 84[-] Evaluating -- loss: 28.43, step: 85[\] Evaluating -- loss: 30.23, step: 86[|] Evaluating -- loss: 39.97, step: 87[/] Evaluating -- loss: 36.87, step: 88[-] Evaluating -- loss: 33.44, step: 89[\] Evaluating -- loss: 31.87, step: 90[|] Evaluating -- loss: 35.19, step: 91[/] Evaluating -- loss: 40.70, step: 92[-] Evaluating -- loss: 32.38, step: 93[\] Evaluating -- loss: 42.29, step: 94[|] Evaluating -- loss: 35.87, step: 95[/] Evaluating -- loss: 31.29, step: 96[-] Evaluating -- loss: 37.70, step: 97[\] Evaluating -- loss: 37.97, step: 98[|] Evaluating -- loss: 32.28, step: 99[/] Evaluating -- loss: 38.07, step: 100[-] Evaluating -- loss: 36.80, step: 101[\] Evaluating -- loss: 30.90, step: 102[|] Evaluating -- loss: 30.51, step: 103[/] Evaluating -- loss: 28.90, step: 104[-] Evaluating -- loss: 26.89, step: 105[\] Evaluating -- loss: 31.31, step: 106[|] Evaluating -- loss: 31.59, step: 107[/] Evaluating -- loss: 28.04, step: 108[-] Evaluating -- loss: 32.58, step: 109[\] Evaluating -- loss: 36.18, step: 110[|] Evaluating -- loss: 39.27, step: 111[/] Evaluating -- loss: 34.87, step: 112[-] Evaluating -- loss: 32.70, step: 113[\] Evaluating -- loss: 26.33, step: 114[|] Evaluating -- loss: 34.51, step: 115[/] Evaluating -- loss: 31.37, step: 116[-] Evaluating -- loss: 23.92, step: 117[\] Evaluating -- loss: 34.13, step: 118[|] Evaluating -- loss: 34.80, step: 119[/] Evaluating -- loss: 39.66, step: 120[-] Evaluating -- loss: 41.73, step: 121[\] Evaluating -- loss: 30.13, step: 122[|] Evaluating -- loss: 31.29, step: 123[/] Evaluating -- loss: 31.11, step: 124[\] Evaluating -- loss: 116.08, step: 14   

Any idea what could cause this?

**Comment by MartinGeierino** [comment]
Restart of RStudio seems to have fiexed this.

**Comment by MartinGeierino** [comment]
It reappeared:

> [1] "EPOCH  6  Train_AVG_Loss:  0.0785963609814644  Test_AVG_Loss:  0.191981390118599"
> [1] "EPOCH  6  Train_AVG_Loss_change:  0.00549710541963577  Test_AVG_Loss_Change:  -0.0142934620380402"
> [/] Training -- loss: 4.42, step: 7442017-12-03 **19:02:26.238852: W C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\platform\cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
> 2017-12-03 19:02:26.238870: W C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\platform\cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
> 2017-12-03 19:03:41.434487: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmprrfnw98r\events.out.tfevents.1512324145.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 19:03:41.434520: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 20 events to C:\Users\Martin\AppData\Local\Temp\tmprrfnw98r\events.out.tfevents.1512324145.DESKTOP-5TN0RO2
> 2017-12-03 19:04:26.014482: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmprrfnw98r\events.out.tfevents.1512324145.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 19:04:26.014497: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 40 events to C:\Users\Martin\AppData\Local\Temp\tmprrfnw98r\events.out.tfevents.1512324145.DESKTOP-5TN0RO2
> 2017-12-03 19:04:26.014881: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:162] The events file C:\Users\Martin\AppData\Local\Temp\tmprrfnw98r\events.out.tfevents.1512324145.DESKTOP-5TN0RO2 has disappeared.
> 2017-12-03 19:04:26.014892: E C:\tf_jenkins\home\workspace\rel-win\M\windows\PY\36\tensorflow\core\util\events_writer.cc:131] Failed to flush 40 events to C:\Users\Martin\AppData\Local\Temp\tmprrfnw98r\events.out.tfevents.1512324145.DESKTOP-5TN0RO2**

**Comment by jjallaire** [comment]
Sorry, I don't have the first idea what could cause this! (seems like a very low level TF logging error)

**Comment by JaneShenYY** [comment]
Have you got this problem solved @MartinGeierino? I encountered the same problem.

**Comment by Iamshg** [comment]
Have you tried to set the path to another driver ,like ```E:\logs``` that Tensorboard writes to . @MartinGeierino @JaneShenYY
